test/test_conditional_feature_importance.py

```diff
@@ -568,3 +569,135 @@ def test_groups_warning(self, data_generator):
             " number of features for which importance is computed: 4",
         ):
             cfi.importance(X, y)
+
+
+class TestCFIReproducibility:
```

Comment:
I would like to avoid classes in tests by all means, to make tests as flat as possible. I think that a class here is not bringing enough.